notebooks/DM-31046-Zooniverse-cutouts.ipynb

```diff
@@ -0,0 +1,700 @@
+      "text/plain": [
+       "<IPython.core.display.Image object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "files = glob.glob(os.path.join(config.outputPath, \"images\", \"*.png\"))\n",
```

Comment:
Consider factoring the file output step out of the task, just like the DB input is. This will keep you from having to reload the data you just generated...

Reply:
The purpose of the task is to make the PNGs for use with zooniverse (generally by putting them somewhere they can be served via http). With typical data, it would make tens of thousands of PNG files.